dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/content/util/ContentUtils.java

```diff
@@ -674,5 +676,34 @@ public static long count(String query, User user, String tmDate) {
         return getPullResults(relationship, contentletIdentifier, condition, limit, offset, sort,
                 user, tmDate, pullParents, -1, null);
     }
+
+	public static String addDefaultsToQuery(String query, final boolean editOrPreviewMode, final
+			HttpServletRequest request){
+		String q = "";
+
+		if(query != null)
```

Comment:
Use a `StringBuilder` instead.

Reply:
agree

Reply:
Java optimizes this under the covers. It's not an advantage anymore.